webapp/src/main/webapp/src/app/sender/archive/sender-archive.component.ts

```diff
@@ -0,0 +1,71 @@
+import { Component, OnInit } from '@angular/core';
+import { select, Store } from '@ngrx/store';
+import { Subscription } from 'rxjs';
+import { BaseListingComponent } from 'src/app/shared/base-listing.component';
+import { SearchEvent } from './search-event';
+import { SenderSyncArchive } from './sender-archive';
+import { SenderArchiveService } from './sender-archive.service';
+import { SenderArchivedLoaded} from './state/sender-archive.actions';
+import { GET_SYNC_ARCHIVE } from './state/sender-archive.reducer';
+
+@Component({
+	selector: 'app-sender-archive',
+	templateUrl: './sender-archive.component.html',
+})
+export class SenderArchiveComponent extends BaseListingComponent implements OnInit {
+
+	count?: number;
+
+	loadedSubscription?: Subscription;
+
+	senderArchiveItens?: SenderSyncArchive[];
+
+	searchEvent: SearchEvent = new SearchEvent;
+
+	constructor(private service: SenderArchiveService,
+		private store: Store) {
+		super();
+	}
+
+	ngOnInit() {
+		this.init();
+
+		this.loadedSubscription = this.store.pipe(select(GET_SYNC_ARCHIVE)).subscribe(
+			countAndItems => {
+				this.count = countAndItems?.count;
+				this.senderArchiveItens = countAndItems?.items
+				this.reRender();
+			}
+		);
+
+		this.dtOptions = {
```

Comment:
If this code is really needed, you need to just set the extra properties you wish to set instead of replacing the whole object to preserve code in the init method in the base class, and this should happen after the call to init()

Reply:
i discard that